app/views/users/new.html.erb

```diff
@@ -1,28 +1,28 @@
 <h1>New User Registration</h1>
 
```

Comment:
So we changed form_for back to form_tag for the moment to get the functionality up and running, refactor in to a form_for once everything is working later on

Reply:
looks good!